src/migrations/hotbar-store/index.ts

```diff
@@ -2,8 +2,10 @@
 
 import version500alpha0 from "./5.0.0-alpha.0";
 import version500alpha2 from "./5.0.0-alpha.2";
+import version500beta4 from "./5.0.0-beta.4";
 
 export default {
   ...version500alpha0,
-  ...version500alpha2
+  ...version500alpha2,
+  ...version500beta4
```

Comment:
Please add a comma here so that future diffs are smaller. (I am thinking of adding a lint for this)